lib/foreman_maintain/cli/packages_command.rb

```diff
@@ -60,12 +66,21 @@ def execute
       subcommand 'is-locked', 'Check if update of packages is allowed' do
         interactive_option(['assumeyes'])
         def execute
-          locked = ForemanMaintain.package_manager.versions_locked?
-          puts "Packages are#{locked ? '' : ' not'} locked"
-          exit_code = locked ? 0 : 1
-          exit exit_code
+          run_scenario_or_rescue do
+            locked = ForemanMaintain.package_manager.versions_locked?
+            puts "Packages are#{locked ? '' : ' not'} locked"
+            exit_code = locked ? 0 : 1
+            exit exit_code
+          end
         end
       end
+
+      def run_scenario_or_rescue
+        yield
+      rescue NotImplementedError
```

Comment:
which commands are not implemented right now?

Reply:
foreman-maintain packages lock/unlock/status/is-locked are not implemented, those are dependent on the foreman-protector plugin.